### PR TITLE
EvseManager: Move the start signed meter values to the TransactionFin…

### PR DESCRIPTION
**Pull request by dorezyuk**
…ished

### CODE REVIEW

File: modules/EvseManager/Charger.cpp [hunk 1]
```diff
@@ -1068,6 +1068,7 @@ bool Charger::startTransaction() {
     stop_transaction_id_tag.clear();
     transaction_active = true;
 
+    signalEvent(types::evse_manager::SessionEventEnum::TransactionStarted);
```
Comment (golovasteek) [inline modules/EvseManager/Charger.cpp:1071]: Maybe add a comment, about power meter slowness here?

File: types/evse_manager.yaml [hunk 2]
```diff
@@ -200,7 +197,10 @@ types:
         description: Transaction finished meter value
         type: object
         $ref: /powermeter#/Powermeter
-      signed_meter_value:
+      start_signed_meter_value:
+        description: Signed meter value
+        type: string
+      stop_signed_meter_value:
         description: Signed meter value
```
Comment (golovasteek) [inline types/evse_manager.yaml:204]: And say maybe on the interface level as well, why both meter values are at the "stop_transaction" event.

**Review by golovasteek — CHANGES_REQUESTED** [review]
Looks good, can yo add comments about rationale into the code?

**Comment by ju4nlu** [comment]
As discussed in person not ideal but I think it's good enough to unblock us for now. I also think that some comments in the code will help us in the future

**Pull request merged**